### PR TITLE
branch step

### PR DESCRIPTION
**Pull request by aloncarmel**

### CODE REVIEW

File: steps/change_code.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ startFlow:
           - text: "Now that you have created the CI/CD pipeline, let's make a change to the code and see the pipeline in action."
           - text: "We will make a small change to the style. Let's change the background color of the calculator results bar."
           - text: "So, :instruction[open the `public/default.css` file and change the **`#result`** CSS selector `background-color` to `#5362e0`]"
-          - text: "Once you're done, open a pull request and you should see your action running. I'll be here to review it."
+          - text: Once you're done, :instruction[create a new branch and open a PR with your changes, and I'll review it].
```
Comment (darmalovan) [inline steps/change_code.yml:15]: didn't we already created a branch?

File: steps/create_new_branch.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+id: create_new_branch
```
Comment (darmalovan) [inline steps/create_new_branch.yml:1]: I think the step is missing from the quest.yaml

File: steps/change_code.yml (hunk 1 above)
Comment (aloncarmel) [reply steps/change_code.yml:15]: Does it matter if we created one already? Arent we supposed to create a new branch for each PR?

Comment (aloncarmel) [reply steps/change_code.yml:15]: @shem8 what do you say?

File: steps/create_new_branch.yml (hunk 2 above)
Comment (aloncarmel) [reply steps/create_new_branch.yml:1]: added

File: steps/change_code.yml (hunk 1 above)
Comment (shem8) [reply steps/change_code.yml:15]: Yeah, it's better they'll open a new branch with every PR

**Review by shem8 — APPROVED** [review]

**Pull request merged**